core/lib/health_check/src/lib.rs

```diff
@@ -15,24 +15,27 @@ pub enum HealthStatus {
     NotReady,
     /// Component is ready for operations.
     Ready,
+    /// Component has encountered a warning. The component is still considered healthy.
+    Warning,
```

Comment:
Not sure this is the best term for such a status. It's currently used in `SyncState` health check (if the EN is out of sync) and in consistency checker (if any L1 batches have mismatched commitment data on L1).

Reply:
"Affected" maybe?